kinta-lib/src/main/kotlin/com/dailymotion/kinta/integration/appgallery/ProgressRequestBody.kt

```diff
@@ -0,0 +1,50 @@
+package com.dailymotion.kinta.integration.appgallery
```

Comment:
That isn't AppGallery specific, is it? Maybe move it to `com.dailymotion.kinta.helper` ?

Reply:
👍